setup.py

```diff
@@ -48,18 +48,19 @@ def get_extras_require() -> str:
             "doc8",
             "scipy",
             "pillow",
-            "pettingzoo>=1.17",
+            "pettingzoo>=1.22",
             "pygame>=2.1.0",  # pettingzoo test cases pistonball
             "pymunk>=6.2.1",  # pettingzoo test cases pistonball
             "nni>=2.3,<3.0",  # expect breaking changes at next major version
             "pytorch_lightning",
+            "gym>=0.22.0",
```

Comment:
Probably add `shimmy` here?

Reply:
Yes, thanks for spotting that :) Also I should increase the required gymnasium version to >=0.26.0 I don't think there are any earlier releases but just to be sure